crates/turbo-tasks-fs/src/rope.rs

```diff
@@ -508,12 +515,9 @@ impl AsyncRead for RopeReader {
 }
 
 impl BufRead for RopeReader {
-    /// Returns the full buffer without coping any data. The same bytes will
-    /// continue to be returned until [consume] is called to either consume a
-    /// partial amount of bytes (in which case the Bytes will advance beyond
-    /// them) or the full amount of bytes (in which case the next Bytes will be
-    /// returned).
     fn fill_buf(&mut self) -> IoResult<&[u8]> {
```

Comment:
I changed / removed these comments because the rustdocs on the traits were better (in my opinion) and more useful from the perspective of the user of the api

Reply:
No strong preference